recipes/DeforumSD.py

```diff
@@ -220,26 +373,56 @@ def related_workflows(self) -> list:
     def render_form_v2(self):
         animation_prompts_editor()
 
-        col1, col2 = gui.columns(2)
+        col1, col2 = gui.columns([1, 11], responsive=False)
         with col1:
-            gui.slider(
-                """
-                #### Frame Count
-                Choose the number of frames in your animation.
-                """,
-                min_value=10,
-                max_value=500,
-                step=10,
-                key="max_frames",
+            gui.text_input(
+                label="",
+                key="max_seconds",
+                min_value=0,
+                step=0.1,
+                value=str(
+                    frames_to_seconds(
+                        gui.session_state.get("max_frames", 100),
+                        gui.session_state.get("fps", 12),
+                    )
+                ),
+                style={
+                    "width": "56px",
+                    "padding": "0.5rem",
+                },
             )
-            gui.caption(
-                """
-Pro-tip: The more frames you add, the longer it will take to render the animation. Test your prompts before adding more frames.
-            """
+            gui.session_state["max_frames"] = seconds_to_frames(
+                gui.session_state.get("max_seconds", 10),
+                gui.session_state.get("fps", 12),
             )
+        with col2:
+            gui.caption("Total seconds")
+
+        gui.write("#### Step 2: Increase Animation Quality")
+        gui.caption(
+            """
+            Once you like your keyframes, increase your frames per second for high quality
+            """
+        )
+        fps_options = [2, 10, 24]
+        option_descriptions = ["Draft", "Stop-motion", "Film"]
+        options = {
```

Comment:
Avoid repeating, do options = {2: "draft: 2 fps", ...} and then use the keys in options and values for format_func